pkg/volume/local/local.go

```diff
@@ -348,6 +348,7 @@ func (dm *deviceMounter) mountLocalBlockDevice(spec *volume.Spec, devicePath str
 	return nil
 }
 
+//lint:ignore SA4009 devicePath is overwritten by design, see comment below
```

Comment:
I think it is actually fine to fix this linter error by passing in
`spec.PersistentVolume.Spec.Local.Path` directly into `mountLocalBlockDevice`. 

golang is pass by value so the caller of `MountDevice` can't see any modified values.

Reply:
That's right, changed it to do so. Thanks!